libs/movie/src/lib/movie/components/header/header.component.scss

```
@@ -6,15 +6,15 @@ header {
   background-size: cover;
 
   .overlay {
-    padding: 24px 24px 48px 24px;
+    padding: 48px 24px 48px 24px;
```

I think you can do then
```suggestion
    padding: 48px 24px;
```